src/components/Accordion/Accordion.test.tsx

```diff
@@ -56,3 +56,18 @@ describe('Accordion', () => {
     expect(stoplightIcon).toHaveClass('bsds-accordion-stoplight-green');
   });
 });
+
+it('Panel should have class "bsds-accordion-expanded" when showing on click of accordion trigger', async () => {
```

Comment:
Could we also add a test to make sure we aren't tabbing into the accordion panel when closed? Create a dummy accordion with a button in the panel, focus the trigger, tab, check that next trigger is focused. There is a similar example in [DropdownMenu (Line 142)](https://github.com/bos-sci/anatomy-react/blob/develop/src/components/DropdownMenu/DropdownMenu.test.tsx#L142) using arrow keys instead of tab.